Templates/Per Tenant Extension/.AL-Go/Current.settings.json

```diff
@@ -0,0 +1,6 @@
+{
+    "artifact":  "////latest",
```

Comment:
is  ////  needed

Reply:
artifact definition is storageaccount/type/version/country/select